### ISSUE TITLE
[Bug] Version 4.8.1 works, but charts seem to be broken using this helm chart

### ISSUE THREAD

**Issue by belmeopmenieuwesim** [RocketChat/helm-charts#63]
I was running version 4.7.4 using this helm chart fine. Then I manually upgraded to 4.8.1 using custom values.yaml:
```yaml
image:
  tag: 4.8.1
```
Everything works fine, except that charts will not render, possible due to the following js error:
![image](https://user-images.githubusercontent.com/52761225/174609517-afe814fc-fea5-4e64-a063-e4ce873a69e7.png)

I also noticed the Github Workflow Action is failing for the latest version.

**Comment by prppedro** [comment]
Yes... There is something wrong with 4.8.1, reason why I didn't release the helm for that version yet. I had a myriad of issues in doing so. Some images weren't available and, for some reason, not even the monolith on most environments. I'm yet to understand why. What environment are you running it on?

**Comment by belmeopmenieuwesim** [comment]
> Yes... There is something wrong with 4.8.1, reason why I didn't release the helm for that version yet. I had a myriad of issues in doing so. Some images weren't available and, for some reason, not even the monolith on most environments. I'm yet to understand why. What environment are you running it on? 

Okay that explains. Too bad they don't support db migration rollbacks 😅. I am running it on a baremetal kubernetes cluster running Ubuntu and containerd.

**Comment by belmeopmenieuwesim** [comment]
I made a PR to at least fix this Chart.js constructor bug: https://github.com/RocketChat/Rocket.Chat/pull/26022, maybe this fixes something?
